django_project/base/views/project.py

```diff
@@ -184,7 +184,8 @@ def get_queryset(self):
             return projects_qs.filter(creator=self.request.user)
 
     def get_context_data(self, **kwargs):
-        context = super(PendingProjectListView, self).get_context_data(**kwargs)
+        context = super(PendingProjectListView,
```

Comment:
Pull left applies here. Please see our coding standards doc in InaSAFE for more details.